src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShippingMethodExampleFactory.php

```diff
@@ -129,6 +142,9 @@ protected function configureOptions(OptionsResolver $resolver): void
             ->setDefined('shipping_category')
             ->setAllowedTypes('shipping_category', ['null', 'string', ShippingCategoryInterface::class])
             ->setNormalizer('shipping_category', LazyOption::findOneBy($this->shippingCategoryRepository, 'code'))
+            ->setDefined('tax_category')
+            ->setAllowedTypes('tax_category', ['null', 'string', TaxCategoryInterface::class])
+            ->setNormalizer('tax_category', LazyOption::findOneBy($this->taxCategoryRepository, 'code'))
```

Comment:
This will fail if `$this->taxCategoryRepository` is null (not provided in the constructor). We need to provide a BC layer for that, either by using a noop repository or a different normalizer.

Reply:
Or we do that in the `create` method and don't normalize this value.

Reply:
Added that.